Hearthstone Deck Tracker/Utility/RemoteConfig.cs

```diff
@@ -135,6 +138,12 @@ internal class BobsBuddyData
 				[JsonProperty("log_lines_kept")]
 				public int LogLinesKept { get; set; }
 			}
+
+			internal class UpdateData
+			{
+				[JsonProperty("image_update_hearthstone_version")]
+				public string Version { get; set; }
```

Comment:
We should just make this a `int` value that we increment as needed. We a) don't want to maintain this version with every hearthstone patch and b) might want to update it multiple times per hearthstone patch is something goes wrong with the images for some reason.